src/main.cc

```diff
@@ -721,8 +721,22 @@ main(int argc, char **argv)
 		script = argv[0];
 		title_set_script_name(argv[0]);
 	}
+	strlcpy(tarantool_path, tarantool_bin, sizeof(tarantool_path));
+	if (strlen(tarantool_path) < strlen(tarantool_bin))
+		panic("executable path is trimmed");
 
-	crash_init(tarantool_bin);
+	/*
+	 * We need to keep clock data locally to report uptime without binding
+	 * to libev and etc. Because we're reporting information at the moment
+	 * when crash happens and we are to be independent as much as we can.
+	 */
+	struct timespec ts;
+	if (clock_gettime(CLOCK_MONOTONIC, &ts) == 0)
+		box_start_time = ts.tv_sec;
+	else
+		say_syserror("fail to get box_start_time, ignore");
```

Comment:
Nit: Let's change to `panic`? We don't expect a failure here, and if the clock doesn't work, we'll probably panic somewhere else anyway.

Reply:
Agreed to leave as is, because the old code used `pr_syserr`.

Let's use a more human-readable error message (without reference to `box_start_time`).

Reply:
Done. `"failed to get start time, ignore"`